src/Controller/BlogController.php

```diff
@@ -57,6 +57,19 @@ public function tag(string $tag, int $page = 1, int $perPage = 20): Response
         ])->setLastModified(ContentUtils::max($pageArticles, 'lastModifiedOrCreated'));
     }
 
+    #[Route('/rss.xml', name: 'blog_rss')]
```

Comment:
Est-ce que ça fait sens d'avoir le RSS dans le sitemap ?

Si non: https://stenopephp.github.io/Stenope/build-configuration/#excluding-a-route-from-the-sitemap